src/main/java/com/rultor/agents/ecs/Amazon.java

```diff
@@ -27,33 +27,34 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package com.rultor.agents.ec2;
+package com.rultor.agents.ecs;
 
-import com.amazonaws.services.ec2.model.Instance;
+import com.amazonaws.services.ecs.model.Container;
 import com.jcabi.aspects.Immutable;
 import java.io.IOException;
 
 /**
- * Amazon EC2 abstraction.
+ * Amazon ECS abstraction.
  *
  * @author Yuriy Alevohin (alevohin@mail.ru)
  * @version $Id$
  * @since 2.0
- * @todo #629 Add implementation for com.rultor.agents.ec2.Amazon.
- *  It must create com.amazonaws.services.ec2.AmazonEC2 client
+ * @todo #629 Add implementation for com.rultor.agents.ecs.Amazon.
+ *  It must create com.amazonaws.services.ecs.AmazonECSClient
  *  with config params "credentials", "zone", "type", "key". Use
- *  client.runInstances(com.amazonaws.services.ec2.model.RunInstancesRequest)
- *  to run on-demand instance. Method runOnDemand must finally
+ *  client.runInstances
+ *  (com.amazonaws.services.ecs.model.RegisterContainerInstanceRequest)
```

Comment:
@xupyprmv I would suggest to simply delete the this sentence ("Use client.runInstances ...") and the one following it ("Method runOnDemand ...") from the Puzzle.
Let's just improve the Javadoc a little to make sure the details come across correctly here and make it:

```java
    /**
     * Run ECS OnDemand container with running SSH- and Docker-Daemon.
     * @return ECS Container with running SSH- and Docker-Daemon
     * @throws IOException if fails
     */
```

This implies that a properly running container is to be returned and adds no confusing details imo.
Also the whole "wait" thing is much less of an issue with ECS compared to EC2, I see how this needed explicit mentioning and probably also some trickery in the implementation for EC2. In ECS this is just seconds anyhow :)